### ISSUE TITLE
Web Player doesn't work

### ISSUE THREAD

**Issue by camwes** [rdio/hello-web-playback#6]
When attempting to play using the web playback API javascipt version the following error occurs:

``` js
Uncaught TypeError: Cannot read property 'rdio_play' of null
3hello.js:44 Uncaught TypeError: Cannot read property 'rdio_play' of null(anonymous function) @ hello.js:44d.event.handle @ jquery-1.5.1.min.js:16k.handle.m @ jquery-1.5.1.min.js:16
```

**Comment by dasevilla** [comment]
Did you generate a playback token for your domain?

http://www.rdio.com/developers/docs/web-playback/overview/ref-playback-model
http://www.rdio.com/developers/docs/web-service/methods/playback/ref-getplaybacktoken

**Comment by camwes** [comment]
yes @dasevilla, it looks like the we playback API was deprecated with the oAuth update, there is apparently a beta somewhere out there: https://groups.google.com/forum/#!topic/rdio-api/_UlW2Oc6Dvc

**Comment by dasevilla** [comment]
The Web Playback API is not yet deprecated, however there is a [JS API beta](https://groups.google.com/d/msg/rdio-api/_UlW2Oc6Dvc/uGhwZUwxSesJ) that provides playback (and more).

You can try using the [Playback Console](http://rdio.github.io/playback-console/). For example, you can enter `rdio_play('t2714868')` into the console. If you're in the US and signed in as a subscriber, you'll hear a full track.

**Comment by camwes** [comment]
think that link is the same one I provided, and I have already sent an email requesting beta access. I am not sure how to get fix the issue that I am experiencing using the web playback APi in my app. It gets the same rdio_play is null error that is listed above. Any idea how to dix that? I have a playback token and nothing has changed in my app on the client since the oauth 2 migration
